src/bionics.cpp

```diff
@@ -2469,10 +2473,33 @@ void bionic::toggle_auto_start_mod()
     if( info().fuel_opts.empty() ) {
         return;
     }
-    if( !has_flag( "AUTO_START_ON" ) ) {
-        set_flag( "AUTO_START_ON" );
+    if( !has_var( "AUTO_START" ) ) {
+        uilist tmenu;
+        tmenu.text = _( "Chose Start Power Level Threshold" );
+        tmenu.addentry( 1, true, 'o', _( "No Power Left" ) );
+        tmenu.addentry( 2, true, 't', _( "Bellow 25 percent" ) );
```

Comment:
I'm pretty sure "Bellow" is a typo, it means to shout, the proper word is "Below".